docs/how-do-i-install-my-operator-with-olm.md

```diff
@@ -1,6 +1,17 @@
-# How do I install my operator with OLM? 
+# How do I install my operator with OLM?
+
+[Once you've made your packaged operator available in a catalog](packaging-an-operator.md) it will appear in the `packagemanifest` list from which the Operators to install are selected. See [How do I list available Operators](list-available-operators.md#information-relevant-for-installation) how to retrieve the required information from the `PackageServer` in order start an installation of an Operator.
+
+## Install with automatic updates enabled
+
+First, retrieve the package name, channel and catalog name and catalog source namespace from the `packagemanifest` of the desired Operator to install.
+
+```bash
+$ kubectl describe packagemanifest my-operator
+```
+
+ With this information a `Subscription` object can be created. This represents the intent to install an Operator from a particular catalog and keep it updated throughout the life cycle of the Operator via newly released version that got subsequently added to the referenced catalog.
```

Comment:
I would recommend changing

>This represents the intent to install an Operator from a particular catalog and keep it updated throughout the life cycle of the Operator via newly released version that got subsequently added to the referenced catalog.

to

>A subscription represents the desire to install an Operator from a particular catalog and keep it updated as new versions are subsequently added to the referenced catalog.

or somethings along those lines.

Reply:
Maybe even 
> With this information, a `Subscription` to an operator can be created.

instead of 

> With this information a `Subscription` object can be created. 

And in @awgreene 's suggestion:

> A subscription represents the desire to install an Operator from a particular catalog and keep it updated as new versions are subsequently added to the referenced catalog.

if we tweak it to

> A subscription represents the desire to install an Operator from a particular catalog, and subscribe to updated version of the operators as they are subsequently added to the catalog.

then if could read like more of a user facing doc than dev facing doc.